CDCgov/prime-reportstream
# 7051 Universal Pipeline Quality Filtering Results

This PR makes various updates to ensure that the data that you fetch from the History API displays correctly through the Quality filtering step for the Universal Pipeline.

Test Steps:
1. On your `organizations.yml` file, make a change on a `qualityFilter:` entry that you can send reports to. Note: This PR enables `ignore-FULL_ERL` as a receiver for testing purposes. The quality filter section for this receiver also includes a commented out example of a filter that will always trigger.
2. Submit a Full ELR report via Postman (or whichever you use)
3. After receiving a `submissionId`, use that to check the History API until the response includes a list of `destinations`.
4. On receiving a response, check that `itemCount` is lower than `itemCountBeforeQualityFiltering` and that there's data on `filteredReportRows` and `filteredReportItems`
5. Revert the changes on Step 1, then repeat steps 2-4. This time `itemCount` should match `itemCountBeforeQualityFiltering`, and `filteredReportRows` and `filteredReportItems` should be empty.

## Changes
- Added `ignore-FULL_ELR` receiver with filters for testing
- Added logging for the Universal Pipeline during the `route` step
- Updated `SubmissionHistory` to handle the different structure of the translation steps on the pipeline process.

## Linked Issues
- #7051

## To Be Done
- #7052
